build/helper.ts

```diff
@@ -19,6 +19,14 @@ const reDocUrl: InstallOptions['reDocUrl'] = (fileName, header) => {
   return docs + fileName + (_header ? `#${_header}` : '')
 }
 
+const reWebTypesSource: InstallOptions['reWebTypesSource'] = (title) => {
+  const symbol = `EL${title.replace(/-/, ' ').replace(/^\w|\s+\w/g, (item) => {
```

Comment:
Should the symbol be started with `El`? I think `EL` seems wrong to me.

Reply:
Yes this is a typo